packages/fast-components-react-base/src/slider/slider.props.ts

```diff
@@ -0,0 +1,112 @@
+import * as React from "react";
+import {
+    ManagedClasses,
+    SliderClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { SliderState, SliderThumb } from "./slider";
+
+export interface SliderManagedClasses extends ManagedClasses<SliderClassNameContract> {}
+export interface SliderUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export enum SliderOrientation {
+    horizontal = "horizontal",
+    vertical = "vertical",
+}
+
+export enum SliderMode {
+    singleValue = "singleValue",
+    adustUpperValue = "adustUpperValue",
+    adustLowerValue = "adjustLowerValue",
+    adjustBoth = "adjustBoth",
+}
+
+export interface SliderRange {
+    minValue: number;
+    maxValue: number;
+}
+
+export interface SliderHandledProps extends SliderManagedClasses {
+    /**
+     * The slider orientation, default is horizontal
+     */
+    orientation?: SliderOrientation;
+
+    /**
+     * The slider mode
+     */
+    mode?: SliderMode;
+
+    /**
+     * Custom function to render the thumb of the control
+     */
+    thumb?: (
+        props: SliderProps,
+        state: SliderState,
+        mouseDownCallback: (event: React.MouseEvent) => void,
+        keyDownCallback: (event: React.KeyboardEvent) => void,
+        thumb: SliderThumb
+    ) => React.ReactNode;
+
+    /**
+     * The initial value of the slider
+     */
+    initialValue?: number | SliderRange;
+
+    /**
+     * The total range of values represented in the slider
+     */
+    range?: SliderRange;
+
+    /**
+     * Constrains the slider to a subset of the whole bar
+     */
+    constrainedRange?: SliderRange;
+
+    /**
+     * Contrains the slider to a particular value increments
+     */
+    step?: number;
+
+    /**
+     * The amount added/subtracted to the current value per page up/down keystroke
+     */
+    pageStep?: number;
```

Comment:
is this different from step?

Reply:
Yes, this is the value for the bigger increment used when people use the page up/down keys.  Step constrains the values to specific increments and also defines the increment used when moving thumbs with arrow keys.